src/Process/FeatureProcessing.js

```diff
@@ -90,6 +90,7 @@ export default {
                 group.layer = layer;
                 group.matrixWorld.copy(node.matrixWorld).invert();
                 group.matrixWorld.decompose(group.position, group.quaternion, group.scale);
+                group.position.z += node.geoidHeight;
```

Comment:
it's necessary?

Reply:
I believe it is, since this `result` group is positioned independently from the `node`.